pages/apim/3.x/kubernetes/apim-kubernetes-operator-definitions.adoc

```diff
@@ -0,0 +1,163 @@
+[[apim-kubernetes-operator-definitions]]
+= Custom Resource Definitions (CRDs)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_definitions.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+
+== Custom Resource Definitions (CRDs) reference
+
+The Gravitee Kubernetes Operator comes with two Custom Resource Definitions (CRDs) - `ApiDefinition` and `ManagementContext`.
+
+=== `ApiDefinition`
+
+The `APIDefinition` custom resource represents the configuration for a single proxied API and its versions. It is similar to a YAML representation of an API Definition in JSON format.
+
+Resources:
+
+  * The `ApiDefinition` CRD code is available on link:https://github.com/gravitee-io/gravitee-kubernetes-operator/blob/alpha/api/v1alpha1/apidefinition_types.go[GitHub^].
+  * The `ApiDefinition` CRD API reference is documented link:{{ '/apim/3.x/apim_kubernetes_operator_api_reference.html#apidefinition' | relative_url }}[here].
+  * You can learn how to use the `ApiDefinition` resource in link:{{ '/apim/3.x/apim_kubernetes_operator_user_guide_api_definition.html' | relative_url }}[this section].
+
+==== Workflow
+
+The following workflow is applied when a new `ApiDefinition` resource is added to the cluster:
+
+  1. The GKO listens for `ApiDefinition` resources.
+  2. The GKO performs some required changes, such as computing automatically IDs or CrossIDs (for API or Plan).
+  3. The GKO converts the definition to JSON format.
+  4. The GKO compares the definition to the existing definition (via a local check from `ConfigMap`). If something has changed, the GKO updates the `ConfigMap` and pushes the definition to the Management API (if a `ManagementContext` is provided).
+  5. The API Gateway listens for `ConfigMap`s with a specific label.
+  6. The GKO deploys the API to the API Gateway.
```

Comment:
I wonder if we should really talk about configMap. because it's more of an internal detail. or just make a parenthesis about the internal functioning. 

But from a user point of view it's superficial 🤷‍♂️ and confusing

Reply:
I'll just remove the "configMap" step (and in the other page too).